internal/pkg/ccstructs/hub_structs.go

```diff
@@ -0,0 +1,26 @@
+package ccstructs
```

Comment:
Are these really part of ccstructs? I thought Confluent Hub was for CP only.

Reply:
It's not, but I don't have any better place to put it.

Reply:
Moved it to a new pkg called `cpstructs`